### DISCUSSION TITLE
How to hide details on the map

### DISCUSSION THREAD

**Discussion by hypocritical20171014** [visgl/deck.gl#7165] [Q&A]
I want to modify the underlying details of the map, such as hiding map labels, roads, buildings, etc., what should I do?

**Comment by Pessimistress** [comment]
deck.gl does not render base maps. You need to visit your base map library's documentation.

**Reply by hypocritical20171014** [reply]
I'm using Deck.gl as parent tag and react-map-gl's staticMap as child tag, but I didn't find instructions to hide map details in react-map-gl's documentation, but I found that Kepler.gl can do it , how is this achieved?

**Reply by Pessimistress** [reply]
https://docs.mapbox.com/mapbox-gl-js/style-spec/

**Reply by hypocritical20171014** [reply]
I see,thanks! but i find a example from react-map-gl document,it show thats! Well,if I upgrade react-map-gl version to v7.0, Am I not able to use staticmap as the bottom layer of deck.gl, but instead use deck.gl as the bottom layer?

**Reply by ZakiOro** [reply]
@hypocritical20171014  can you explain what you mentioned, as i am using the getting started example, seems the map is covering the deckgl layer.
